app/femr/ui/views/triage/index.scala.html

```diff
@@ -63,6 +63,7 @@ <h2 class="text-center">Check In - Triage</h2>
                     @inputAge("Age", "Years", "years", "Months", "months", if(viewModel != null) viewModel.getPatient else null)
                     <span class="orSpan">OR</span>
                     @inputDate("Birth Date", "age", if(viewModel != null) viewModel.getPatient.getBirth else null)
+                    <label class="checkBoxLabel"><input type="checkbox" name="birthdayIsFake"> BIRTHDAY IS FAKE</label>
```

Comment:
The system should be able to identify if the birthday is fake behind the scenes (this could be a hidden checkbox that gets checked by JavaScript when a birthdate is generated automatically). something similar to what @gerrytucker78 did in the javascript file here:
https://github.com/FEMR/femr/pull/568/files#diff-08ff3973c54207c832f9ae34bff8753cR590


creating extra clicks for a user should only be done if absolutely necessary!